src/ci/bin/test_slurm.inc.sh

```diff
@@ -47,15 +47,15 @@ SlurmctldDebug=3
 SLURM_CONF
 
     # Munge will run as the munge user, so change the owner of the directory
-    sudo chown munge:munge /var/run/munge
-
-    # Create a munge key, required for slurm-llnl
-    dd if=/dev/random bs=1 count=1024 | sudo tee /etc/munge/munge.key >/dev/null
-    sudo chown munge:munge /etc/munge/munge.key
-    sudo chmod 400 /etc/munge/munge.key
-
-    # Start munge as the munge user
-    sudo -u munge munged
+#    sudo chown munge:munge /var/run/munge
+#
+#    # Create a munge key, required for slurm-llnl
+#    dd if=/dev/random bs=1 count=1024 | sudo tee /etc/munge/munge.key >/dev/null
+#    sudo chown munge:munge /etc/munge/munge.key
+#    sudo chmod 400 /etc/munge/munge.key
+#
+#    # Start munge as the munge user
+#    sudo -u munge munged -F
```

Comment:
As the above comment says:

> \# A mash of configure-until-it-runs. Feel free to PR suggestions/fixes

Delete this block if it the rest of the code runs. 👍